agent/Dockerfile.local

```diff
@@ -8,7 +8,7 @@ ADD .bashrc /root/.bashrc
 VOLUME /usr/src/app/data
 RUN apt update && apt install -y vim nano netcat curl nano
 
-ENV DATA_DIR="/data" STREAMSETS_USERNAME="admin" STREAMSETS_PASSWORD="admin" STREAMSETS_URL="http://localhost:18630" \
+ENV DATA_DIR="data" STREAMSETS_USERNAME="admin" STREAMSETS_PASSWORD="admin" STREAMSETS_URL="http://localhost:18630" \
```

Comment:
why?

Reply:
it easier to test and debug when both docker container and local environments share same same destinations, sources and pipelines